index.html

```diff
@@ -0,0 +1,121 @@
+            </div>
+            <div>
+                <label for="toelichting" class="form-label">Toelichting</label>
+                <p class="label-subtitle">Wat is het fictieve probleem dat je wil oplossen met jouw applicatie (het doel).</p>
+                <textarea  id="toelichting" class="form-textarea" rows="5" placeholder="Toelichting over het project"></textarea>
+            </div>
+            <div>
+                <label class="form-label">Kernfunctionaliteiten</label>
+                <div id="kernFunctionaliteitenContainer" class="space-y-2">
+                    <!-- Initial Fields -->
+                    <div class="functionaliteit-item">
+                        <input type="text" class="form-input kern-functionaliteit" placeholder="Functionaliteit 1" required>
+                    </div>
+                    <div class="functionaliteit-item">
+                        <input type="text" class="form-input kern-functionaliteit" placeholder="Functionaliteit 2" required>
+                    </div>
+                    <div class="functionaliteit-item">
+                        <input type="text" class="form-input kern-functionaliteit" placeholder="Functionaliteit 3" required>
+                    </div>
+                </div>
+                <button type="button" id="addKernFunctionaliteit" class="btn-primary mt-2">+</button>
+            </div>
+
+
+
+            <!-- Backend & Fullstack Fields -->
+            <div id="backendFields" class="hidden">
+                <div>
+                    <label for="entities" class="form-label">Entiteiten</label>
+                    <p class="label-subtitle">Wat zijn entiteiten in Java? Een entiteit vertegenwoordigt een object in de database en lijkt op iets zoals je het in de echte wereld ook hebt, zoals gebruikers, producten, adressen, bestellingen, hotelkamers, etc. Deze vertalen we dan naar entiteiten met specifieke eigenschappen, zoals een hotelkamer bijvoorbeeld altijd een kamernummer, aantal slaapplaatsen, of het een roken/niet-roken kamer is en een prijs per nacht. Dit kun je dan als volgt opschrijven: Room(roomNumber, capacity, smokingAllowed, pricePerNightPerPerson)</p>
+                    <textarea id="entities" class="form-textarea" rows="5" placeholder="Elke entiteit op een nieuwe regel"></textarea>
+                </div>
+            </div>
+
+            <!-- Frontend Fields -->
+            <div id="frontendFields" class="hidden">
+                <div>
+                    <label class="form-label">Kies een backend</label>
+
+                    <p class="label-subtitle">Wij faciliteren de <a class="link" target="_blank"  href="https://novi.datavortex.nl">Novi Educational Backend.</a> dit is een database waar je gebruikers kan opslaan. Meer informatie staat op de website. Je mag ook een database gebruiken van eerdere leerlijn of een firebase gebruikers database. Deze zijn lastiger om te configureren en je krijgt er niet een extra punt voor.  </p>
```

Comment:
Vanuit NOVI adviseren we je de [NOVI Educational Backend](https://novi.datavortex.nl/) te gebruiken. Hierin kun je de gebruikersnaam, email, wachtwoord, en profielfoto in opslaan. Er is ook een extra tekstveld beschikbaar om eigen informatie in op te slaan. Als je een specifiek idee hebt waar je meer opslagmogelijkheden voor nodig hebt, heb je de optie om een eigen [Firebase backend](https://firebase.google.com/) op te zetten. Maar let op: deze zijn lastiger om te configureren! Het kost extra tijd en je krijgt er geen extra punten voor. Indien je de Backend leerlijn bij NOVI hebt gevolgd is het ook toegestaan om jouw eigen gemaakte Spring Boot project als backend te gebruiken.